plugins/wazo_fanvil/common/common.py

```diff
@@ -394,7 +395,7 @@ def _format_fkey(
     ) -> FKeyDict:
         fkey: FKeyDict = {'id': funckey_number, 'title': funckey['label'], 'type': 2}
         if funckey['type'] == 'speeddial':
-            fkey['type'] = 4
+            fkey['type'] = 1
```

Comment:
This affects all the Fanvil phones. Are you sure this is the right type? Will it work on phones other than H2U?

Reply:
Ok :smile: thank you!